Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -150,18 +135,16 @@ int main()
 	std::cout << "Your choice: " << Choice << std::endl;
 	if (Choice == 1) 
 	{
-		Circle circle;
-		circle.DataCircle();
+		FCircle circle;
```

Comment:
Remember we use PascalCase. `circle` is not a correct name.

Reply:
Applied